docker/post-install

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+
+dnf install -y /tmp/rpms/*.rpm
+
+# Be nice for docker exec and link SPDK scripts|binaries under common PATH
+# location like /usr/bin.
+ln -sf $(ls -1dpA /usr/local/bin/* | grep -v "/$") /usr/bin
+ln -s /usr/libexec/spdk/scripts/rpc.py /usr/bin
+ln -s /usr/libexec/spdk/scripts/rpc_http_proxy.py /usr/bin
+ln -s /usr/libexec/spdk/scripts/setup.sh /usr/bin
+ln -s /usr/libexec/spdk/include/spdk /usr/include
+ln -s /usr/libexec/spdk/scripts/ /usr
+
+
+dnf clean all
```

Comment:
If we start using `RUN --mount=type=cache,...` it's better not to add the post clean-ups, or the cache will be useless.